data/ka'het/ka'het outfits.txt

```diff
@@ -289,6 +395,27 @@ outfit "Telis Engine Nacelles"
 	"steering flare sound" "plasma medium"
 	description "These nacelles used to comprise the entire engine block of a Telis'het. While their size is not impressive compared to other engines, if used in conjunction with a sustainer or two, they can be a powerful form of propulsion, combining both turning and thrusting engines in one."
 
+outfit "Vareti Engine Block"
```

Comment:
Why "Block" instead of "Nacelles"?

Reply:
Because the sprite isn't nacelles,  it is a block